lib/rbslicer.rb

```diff
@@ -84,85 +84,80 @@ def wrapper_test()
     end
     return base_url
   end
-  # Public: Create field in Slicing Dice API
+  # Public: Create column in Slicing Dice API
   #
-  # query - A Hash in the Slicing Dice field format
+  # query - A Hash in the Slicing Dice column format
   #
   # Examples
   #
-  #    field_data = {
-  #     'name' => 'Rbslicer String Field',
+  #    column_data = {
+  #     'name' => 'Rbslicer String Column',
   #     'description' => 'Ruby Test Description',
   #     'type' => 'string',
   #     'cardinality' => 'low'
   #    }
-  #    puts sd.create_field(field_data)
+  #    puts sd.create_column(column_data)
   #    => {
   #    ...  "status" => "SUCCESS",
-  #    ...  "field_name" => "ruby-string-field"
+  #    ...  "column_name" => "ruby-string-column"
   #    ....}
   #
   # Returns a hash with api result
-  def create_field(query)
+  def create_column(query)
     base_url = wrapper_test()
     if query.kind_of?(Array)
-      query.each { |q| field_create(q) }
+      query.each { |q| column_create(q) }
     else
-      field_create(query)
+      column_create(query)
     end
   end
 
-  def field_create(query)
+  def column_create(query)
     base_url = wrapper_test()
-    sd_validator = Utils::FieldValidator.new(query)
+    sd_validator = Utils::ColumnValidator.new(query)
     if sd_validator.validator
-      url = base_url + METHODS[:field]
+      url = base_url + METHODS[:column]
       make_request(url, "post", 2, data= query)
     end
   end
 
-  # Public: Get all fields created on Slicing Dice API
-  def get_fields()
+  # Public: Get all columns created on Slicing Dice API
+  def get_columns()
     base_url = wrapper_test()
-    url = base_url + METHODS[:field]
+    url = base_url + METHODS[:column]
     make_request url, "get", 2
   end
 
-  # Public: Send a indexation to Slicing Dice API
+  # Public: Send a insertion to Slicing Dice API
   #
-  # data - A Hash in the Slicing Dice field format
-  # auto_create_fields - if true Slicing Dice API will create nonexistent
-  # fields automatically
+  # data - A Hash in the Slicing Dice column format
+  # auto_create_columns - if true Slicing Dice API will create nonexistent
```

Comment:
"auto_create_columns" parameter should be removed.
Check this PR (https://github.com/SlicingDice/slicingdice-ruby/pull/10)